test/classes/initializers/generics/initAtomicsViaWrite.bad

```diff
@@ -1,4 +1,4 @@
-initAtomicsViaWrite.chpl:3: internal error: GEN0433 chpl Version 1.16.0 pre-release (c0bf20b)
```

Comment:
I'd be inclined to leave the .bad files as they were assuming that the script which is papering over SHAs and release numbers works for them (and I'd adjust the script if it doesn't).  Reason being that we should try and minimize our maintenance workload.  Also it'll be nice for the 1.17 release to be able to look at this PR and see which n files need to change at a minimum, and I worry we'd get into the practice of making updates like this needlessly.